modules/oe_content_event/config/install/field.storage.node.oe_event_programme.yml

```
@@ -14,7 +14,7 @@ settings:
 module: entity_reference_revisions
 locked: false
 cardinality: -1
-translatable: false
+translatable: true
```

```suggestion
translatable: false
```
Actually field.field.node.oe_event.oe_event_programme.yml has `translatable: false`, so this field isn't translatable.

---

Right